scripts/shared/lib/debug_functions

```diff
@@ -13,7 +13,7 @@ function trap_commands() {
           ! [[ "$cmd" =~ ^(echo|read|\[|while|for|local) ]] &&
           { [[ ! "$cmd" =~ \$\( ]] || cmd="${cmd@Q}"; } &&
           ctxt="${cluster:+[${cluster}]}" &&
-          cmd=`eval echo "[${PWD##*/}]\$ $ctxt $cmd" 2>/dev/null` &&
+          cmd=`eval echo "[$(date -u)] [${PWD##*/}]\$ $ctxt $cmd" 2>/dev/null` &&
```

Comment:
The default format only shows the time with second granularity, which might not be sufficient in all cases; it also shows the date, which isn’t useful here.

```suggestion
          cmd=`eval echo "[$(date +%R:%S:%N)] [${PWD##*/}]\$ $ctxt $cmd" 2>/dev/null` &&
```

Reply:
I think ms precision is sufficient, updated.
btw `%N` is not supported on macOS `date` command so we get something like `09:48:01:N`, but I think it's acceptable. We can also use Perl of `gdate` which is cross-platform but I guess we cannot rely on those being installed everywhere the scripts are intended to be executed.

Reply:
Thanks, ms precision is indeed sufficient. The target platform is the Shipyard base image, non-container-based local builds are nice to have but I agree display differences such as this are acceptable. The Shipyard base image doesn’t include Perl.